product_variant_standard_price_update/models/product.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import _, api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    @api.multi
+    def _compute_same_cost_variants(self):
+        for template in self:
+            cost = template.mapped('product_variant_ids.standard_price')
+            if cost.count(cost[0]) == len(cost):
```

Comment:
This will fail if no variant (for example, with the product_variant_configurator module)